src/ImageDeleteModal.jsx

```diff
@@ -5,13 +5,21 @@ import cockpit from 'cockpit';
 
 const _ = cockpit.gettext;
 
+function sortTags(a, b) {
+    if (a.endsWith(":latest"))
```

Comment:
so `RepoTags` really contains the `:` in the names? (That sounds strange)

Reply:
repo tags are the full strings with registry, name and tag.
`[ "docker.io/library/busybox:latest", "docker.io/library/busybox:1", "docker.io/library/busybox:2", "docker.io/library/busybox:3" ]` for example